ajax/libs/can/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "can",
+  "title": "CanJS",
+  "description": "MIT-licensed, client-side, JavaScript framework that makes building rich web applications easy.",
+  "filename": "can.js",
+  "keywords": [
+    "CanJS",
+    "DoneJS"
+  ],
+  "author": {
+    "name": "Bitovi",
+    "email": "contact@bitovi.com",
+    "web": "https://bitovi.com/"
+  },
+  "homepage": "https://canjs.com/",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:canjs/canjs.git",
+    "github": "https://github.com/canjs/canjs"
+  },
+  "license": "MIT",
+  "npmName": "can",
```

Comment:
I could go either way on this. Since NPM is the official distribution mechanism, that made more sense to me. The github repo also does not include the built files, so I'm not sure we could use the git auto-update anyway.